projects/plugins/backup/src/php/class-jetpack-backup.php

```diff
@@ -22,6 +22,8 @@ public function __construct() {
 		// Set up the REST authentication hooks.
 		Connection_Rest_Authentication::init();
 
+		add_action( 'rest_api_init', array( $this, 'register_rest_routes' ) );
```

Comment:
We now have https://github.com/Automattic/jetpack/blob/2a3b7db17d3ec9256aa61cb295f5da08f99245fc/projects/packages/backup/src/class-rest-controller.php (see #20182), that may help you here, and that may be a good place for other endpoints in the future. Maybe that can help here?

Reply:
The location of these was discussed a little in slack.  I think @fgiannar preferred them in the plugin unless they needed to be shared.  It also kept them with the corresponding client calls.  Realistically they could be moved with no impact, so I'm fine either way.

Reply:
That makes sense! It's fine here in this case.

As the main `Jetpack_Backup` class grows it may make sense to move things like the endpoints to their own class and file, but I suppose we don't have to cross that bridge now. :)